scripts/helperscripts/setup_autohotspot.sh

```diff
@@ -14,6 +14,13 @@ if [[ "$#" -lt 2 || ( "${AUTOHOTSPOTconfig}" != "NO" && "${AUTOHOTSPOTconfig}" !
     exit 1
 fi
 
+call_with_apt_packages_from_file () {
+    local package_file="$1"
+    shift
+
+    # read line from the file and remove comments. Pass it over xargs as arguments to the given command.
+    sed 's/#.*//g' ${package_file} | xargs "$@"
```

Comment:
Isn't it better understandable, if you call here the apt-get command instead of using the passed arguments?!

I mean:
`sed 's/#.*//g' ${package_file} | xargs ${apt_get} install`

The name of the function already implies, that something with apt will happen

Reply:
I didn't want to define `apt_get` and `allow_downgrades` again as these are still used in the `install_main ` function. And making them script wide available doesn't seem right either, as its only used in `install_main`.

The function body is used the same way in the `test_installation` script, only with the `echo` command.

Inthe  `test_installation` script it named it "apt" mainly to distinguish from the function for the "pip" packages and carried over the name.

Reply:
renamed the function to `call_with_args_from_file` and added a function usage comment